config/kafka-capacity-config.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 # see https://docs.google.com/spreadsheets/d/1W4H1IgDXKv24Sf7aWeqeAqxjKVj8EJB9giYN9obqPgU/edit#gid=493008265
 ingressEgressThroughputPerSec: "2Mi"
+maxCapacity: 1000
```

Comment:
This was only used for the /status endpoint which has now been [removed](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/824/files). Was this added intentionally? :thinking: